ui/src/app/pages/workspace/research-purpose.tsx

```diff
@@ -124,136 +107,8 @@ export const ResearchPurpose = fp.flow(
-        <div style={styles.sectionSubHeader}>
-          {researchPurposeQuestions[4].header}
-        </div>
-        <div style={{ ...styles.sectionItemWithBackground, padding: '15px' }}>
-          {workspace.researchPurpose.anticipatedFindings}
-        </div>
-      </div>
-
-      {/* Findings section*/}
-      <div style={styles.sectionHeader}>Findings will be disseminated via:</div>
-      <div style={styles.sectionContentContainer}>
-        {workspace.researchPurpose.disseminateResearchFindingList.map(
-          (disseminateFinding, i) => (
-            <div
-              key={i}
-              style={{
-                ...styles.sectionItemWithBackground,
-                marginTop: '0.75rem',
-              }}
-            >
-              {
-                disseminateFindings.find(
-                  (finding) => finding.shortName === disseminateFinding
-                ).label
-              }
-            </div>
-          )
-        )}
-      </div>
-
-      {/* Outcomes section*/}
-      <div style={styles.sectionHeader}>
-        Outcomes anticipated from the research:
-      </div>
-      <div style={styles.sectionContentContainer}>
-        {workspace.researchPurpose.researchOutcomeList.map(
-          (workspaceOutcome, i) => (
-            <div
-              key={i}
-              style={{
-                ...styles.sectionItemWithBackground,
-                marginTop: '0.75rem',
-              }}
-            >
-              {
-                researchOutcomes.find(
-                  (outcome) => outcome.shortName === workspaceOutcome
-                ).label
-              }
-            </div>
-          )
-        )}
-      </div>
-
-      {/* Underserved populations section*/}
-      {workspace.researchPurpose.populationDetails.length > 0 && (
-        <React.Fragment>
-          <div style={styles.sectionHeader}>Population of interest</div>
-          <div style={styles.sectionContentContainer}>
-            <div style={{ marginTop: '0.75rem' }}>
-              {getSelectedPopulations(workspace.researchPurpose)}
-            </div>
-          </div>
-        </React.Fragment>
-      )}
-
-      {/* AI/AN Research Approach section*/}
-      <div style={styles.sectionHeader}>AI/AN Research Approach</div>
-      <div style={styles.sectionContentContainer}>
-        <div style={styles.sectionSubHeader}>
-          {researchPurposeQuestions[10].header}
-        </div>
-        <div style={{ ...styles.sectionItemWithBackground, padding: '15px' }}>
-          {aianResearchTypeMap.get(workspace.researchPurpose.aianResearchType)}
-        </div>
-        <div style={styles.sectionSubHeader}>
-          {researchPurposeQuestions[11].header}
-        </div>
-        <div style={{ ...styles.sectionItemWithBackground, padding: '15px' }}>
-          {workspace.researchPurpose.aianResearchDetails}
-        </div>
-      </div>
+      </FlexRow>
+      <ResearchPurposeSection researchPurpose={workspace.researchPurpose} />
```

Comment:
Replaced most of component with with common component ui/src/app/components/research-purpose-section.tsx